src/transforms/css.ts

```diff
@@ -61,16 +64,25 @@ export function castVariants(property: any, value: any) {
  */
 export function resolveCssCallees(code: string, cb: (ast: ASTNode) => any): any {
   const ast = parseAst(code)
-  let result: any = false
+  let cssDeclaration: any = false
+  let keyFramesDeclaration: any = false
   visitAst(ast, {
     visitCallExpression(path: any) {
       if (path.value.callee.name === 'css') {
-        result = defu(result || {}, cb(path.value.arguments[0]))
+        cssDeclaration = defu(cssDeclaration || {}, cb(path.value.arguments[0]))
       }
       return this.traverse(path)
     },
+    visitVariableDeclaration(path: any) {
+      if (path.value.declarations[0]?.init?.callee?.name === 'keyFrames') {
```

Comment:
Continuation of the naming convention that matches the naming of the corresponding CSS `@keyframes` rule.

```suggestion
      if (path.value.declarations[0]?.init?.callee?.name === 'keyframes') {
```

https://developer.mozilla.org/en-US/docs/Web/CSS/@keyframes